app/parse-siti-payment-file/build-payment-file.js

```diff
@@ -51,10 +23,11 @@ const buildAndTransformParseFile = (fileBuffer, sequence) => {
       const paymentRequests = batch.paymentRequests
       const batchSequence = batch.sequence
       const batchExportDate = batchHeaders?.length ? batchHeaders[0].exportDate : null
+      const sourceSystem = batchHeaders?.length ? batchHeaders[0].sourceSystem : null
```

Comment:
is this a practise we should be encouraging, any calculation/assignment to a variable should result in at least `null` rather than `undefined`?